server/src/test/java/org/elasticsearch/common/xcontent/support/XContentHelperTests.java

```diff
@@ -105,4 +106,104 @@ public void testToXContent() throws IOException {
             }
         }
     }
+
+    public void testChildBytes() throws IOException {
+
+        for (XContentType xContentType : XContentType.values()) {
+
+            // TODO how to deal with embedded objects?
+
+            XContentBuilder builder = XContentBuilder.builder(xContentType.xContent());
+            builder.startObject();
+                builder.startObject("level1");
```

Comment:
When this code is (eventually) reformatted, the indentation for all these `builder` lines will be normalised. If you want to keep the indent, you could use extra braces to create new indent levels. Or you could build it from JSON.

Reply:
I don't particularly mind losing the indent, it was to help me out while writing the test but isn't necessary.  We can't build from JSON because we're trying to test all possible xcontent formats.